src/utils/backend/person/student.ts

```diff
@@ -65,8 +95,8 @@ export async function createStudent(
     },
     body: JSON.stringify({
       email,
-      password: student.birthdate.split("-").join(""),
-      id: createdStudent[0]?.id,
+      password: "mysk2022autumnrelease",
```

Comment:
I don't think you should have the password shown publicly in a public repository. Put it in `.env.local` or something.